vm/src/builtins/int.rs

```diff
@@ -326,7 +326,8 @@ impl PyInt {
 
 #[pyclass(
     flags(BASETYPE),
-    with(PyRef, Comparable, Hashable, Constructor, AsNumber, Representable)
+    with(PyRef, Comparable, Hashable, Constructor, AsNumber, Representable),
+    itemsize(BigInt)
```

Comment:
I don't get how this is intended to be used yet. How to handle it if it requires multiple fields?

Reply:
It was not intended for multiple types to be declared with `itemsize`. 
In the pyclass code, if the number of types declared with itemsize is not one, it is planned to raise an error in a way that developers can understand. Unlike `basicsize`, it seems difficult to automatically infer item type for each class delcaration, so I have chosen this way.

Reply:
Hmm.. I think `itemsize` implementation is better to be moved to other PRs. It seems it does not cause wrong behavior in `best_base`.